### PR TITLE
Add 2019.4 doc link and fix some bits of the process.

### PR DESCRIPTION
**Pull request by patrickvacek**
It seems like every time I do this I find something wrong with the steps.

### CODE REVIEW

File: docs/README.adoc [hunk 1]
```diff
@@ -43,6 +43,7 @@ The link above is for the doxygen docs on master. Doxygen docs for the following
 * https://advancedtelematic.github.io/aktualizr/2019.1/index.html[2019.1]
 * https://advancedtelematic.github.io/aktualizr/2019.2/index.html[2019.2]
 * https://advancedtelematic.github.io/aktualizr/2019.3/index.html[2019.3]
+* https://advancedtelematic.github.io/aktualizr/2019.3/index.html[2019.4]
```
Comment (lbonn) [inline docs/README.adoc]: wrong link :)

Comment (patrickvacek) [reply docs/README.adoc]: Ah! Good catch, fixed. Also rebased.

**Comment by zabbal** [comment]
> It seems like every time I do this I find something wrong with the steps.

Btw, do we have release process/steps documented somewhere?

**Comment by lbonn** [comment]
> > It seems like every time I do this I find something wrong with the steps.
> 
> Btw, do we have release process/steps documented somewhere?

That's exactly what this PR is touching actually. But the steps were slightly incorrect it seems.

**Comment by patrickvacek** [comment]
> > > It seems like every time I do this I find something wrong with the steps.
> > 
> > 
> > Btw, do we have release process/steps documented somewhere?
> 
> That's exactly what this PR is touching actually. But the steps were slightly incorrect it seems.

Exactly. I have a ticket to automate this work, but it's rather low priority because we don't issue new releases too often, so the potential savings aren't great. But still, it would be nice to not have to think about this so much anymore.

**Pull request merged**